wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConfigImpl.java

```diff
@@ -96,6 +103,12 @@ private void addConfigValues(String section, String subsection, Set<String> name
     public Config set(String name, String value) throws GitException {
         ConfigKey key = parseName(name);
         repository.getConfig().setString(key.section, key.subsection, key.name, value);
+        try {
+            this.repository.getConfig().save();
+        } catch (IOException e) {
+            LOG.debug(String.format(ERROR_FAIL_SAVE_CONFIG, e.getCause().getMessage()), e.getCause());
```

Comment:
note: as now you're sending the exception in the GitException (before it was dropped) the logger call is optional